site/content/en/docs/Installation/_index.md

```diff
@@ -45,17 +43,17 @@ Each version of Agones supports a specific version of Kubernetes. When a new ver
 
```

Comment:
"Each version support a specific version of k8s" (line 42) should be updated since now a version will support more than one.